pkg/cmd/openshift/openshift.go

```diff
@@ -81,6 +81,8 @@ func CommandFor(basename string) *cobra.Command {
 		cmd = kubernetes.NewCommand(basename, basename, out)
 	case "origin", "atomic-enterprise":
 		cmd = NewCommandOpenShift(basename)
+	case "pod-diagnostics":
```

Comment:
@smarterclayton approve or disapprove the new symlink/top level command.  I think I'd rather keep a subcommand until we're sure.